app/controllers/facilities_controller.rb

```diff
@@ -37,8 +31,10 @@ def dashboard
 
   # GET /facilities
   def index
-    @facilities = Facility.active.sorted
+    @facilities = Facility.active.alphabetized
+    @recently_used_facilities = MostRecentlyUsedSearcher.new(acting_user).recently_used_facilities.alphabetized
     @active_tab = "home"
+    @recent_products = MostRecentlyUsedSearcher.new(acting_user).recently_used_products.alphabetized
```

Comment:
Are we sure that including the `alphabetized` scope here doesn't affect which products are selected as recently used? I'm a little leery of adding additional sorting in here because of the way active record chains scopes. Can we test this?

Reply:
The `alphabetized` scope adds a secondary column to sort by, in this case the end of the query looks like this `ORDER BY MAX(orders.ordered_at) DESC, lower(products.name)`.  Testing it out it doesn't seem to change the order at all.  In fact, if you take a closer look at the first screenshot above, you'll see the products are not listed in alpha order.  The ordering in the two more recent screenshots comes from the partial, which uses Array#sort on the products collection returned by `@recent_products`.

Reply:
Should we ditch the `alphabetized` scope for products since it isn't doing much?

Reply:
I was just about to suggest that 😀 . Leaving it in seems misleading, since the actual ordering is provided by the Array#sort in the partial.

It might be nice to pull the sort out of the partial and into the controller (and add a comment on why we're doing the sort in ruby and not in SQL)--you'd need to make sure you made the change for everywhere the partial is used, though.

Reply:
Hm, it looks like this would mean putting 10 `.sort`s in the `facilities/show` view.  Is there another way to clarify intent here?  Maybe a code comment in the facilities controller where we call the MostRecentlyUsedSearcher's?